espnet/asr/pytorch_backend/asr.py

```diff
@@ -1430,88 +1425,3 @@ def grouper(n, iterable, fillvalue=None):
             if num_images >= args.num_images and enh_writer is None:
                 logging.info("Breaking the process.")
                 break
-
-
-def ctc_align(args):
```

Comment:
I'm not sure why this function is deleted.

Reply:
I think this function is unused and duplicated (outdated version) of espnet.bin.asr_align.ctc_align
https://github.com/espnet/espnet/blob/67509aeac232ce57dcf2ec6d91d5684ba317ba9b/espnet/bin/asr_align.py#L228

Reply:
cc @lumaku @jnishi is it correct?

Reply:
Yes, it is an older version of `espnet.bin.asr_align.ctc_align`.
The `ctc_align()` in `espnet/asr/pytorch_backend/asr.py` was written by @simpleoier